shell.nix

```
@@ -0,0 +1,55 @@
+let
+  # sources = import ./nix/sources.nix;
+  # pkgs = import sources.nixpkgs {};
+
+  self = import ./. {};
+  inherit (self) sources pkgs;
```

continuing from the comment left on `default.nix`...

Should I do this:

```suggestion
  sources = import ./nix/sources.nix;
  pkgs = import sources.nixpkgs {};

  self = import ./. {};
```

---

Done in https://github.com/stackabletech/operator-templating/pull/323

---

Personally I prefer keeping `default.nix` as the canonical source to import from.

---

Cool, will mark this as resolved.